src/chains/opStack/actions/portalVersion.ts

```diff
@@ -0,0 +1,109 @@
+// TODO replace this import with copy pasting the abi in
+import { OptimismPortal2Abi as portalAbi } from '@tevm/opstack'
+import {
+  type ReadContractErrorType,
+  readContract,
+} from '~viem/actions/index.js'
```

Comment:
```suggestion
} from '~viem/actions/index.js'
```
Guessing this should be a relative path?

Reply:
yep, and not pointing at `index.js` (should be the actual module)